prime-router/settings/organizations.yml

```diff
@@ -3226,6 +3226,8 @@
         suppressHl7Fields: PID-39-1, PID-39-2, PID-39-3
         replaceValueAwithB:
           MSH-3: [ "*": "CDC PRIME - Atlanta, Georgia (Dekalb)^2.16.840.1.114222.4.1.237821^ISO" ]
+          MSH-5: [ "*": "WADOHPHRED^2.16.840.1.113883.3.237.4.2^ISO" ]
+          MSH-6: [ "*": "dn1fro00^1.3.6.1.4.1.38630.2.1.1.19^ISO"  ]
           MSH-11-1: [ "*": "D" ]
```

Comment:
I know you didn't include this line here, but I think we should take this line out so we don't run into problems when generating HL7 messages for WA. (MSH-11-1)